libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -215,7 +215,7 @@ export class SkyAgGridWrapperComponent
         agGridClassObserver.disconnect();
       });
     }
-    setTimeout(callback);
+    callback();
```

Comment:
what changed that we don't need the `setTimeout` anymore?

Reply:
By using the observables rather than making the changes directly, the `setTimeout` was no longer needed.